common/lib/capa/capa/templates/optioninput.html

```diff
@@ -1,7 +1,7 @@
 <% doinline = "inline" if inline else "" %>
 
 <form class="inputtype option-input ${doinline}">
-   <select name="input_${id}" id="input_${id}" aria-label="${label}" aria-describedby="answer_${id}">
+   <select name="input_${id}" id="input_${id}" aria-label="${response_data['label']}" aria-describedby="answer_${id} ${response_data['description_ids']}">
```

Comment:
Why does this template use both response_data['label'] and response_data['description_ids']? It looks like the other templates use one or the other (I think based on whether or not they are "fully accessible").